src/main/java/org/shikshalokam/backend/elevateUtility/DeleteUserSAAS.java

```
@@ -0,0 +1,103 @@
+
+        Response response = given()
+
+                .contentType("application/x-www-form-urlencoded; charset=UTF-8")
+                .formParam("identifier", loginId)
+                .formParam("password", password)
+                .header("Origin", "qa.elevate-shikshagraha.shikshalokam.org")
+                .post("user/v1/account/login");
+
+        if (response.getStatusCode() != 200) {
+            throw new RuntimeException("Login failed → " + response.getStatusCode());
+        }
+
+        //Extract userId from response (example path)
+        User_ID = response.jsonPath().getString("result.user.id");
+    }
+
+
+    //Log in to Admin
+    public static void loginTOAdmin (String loginId, String password)
+    {
+        //https://qa.elevate-apis.shikshalokam.org/user/v1/admin/login
+
+        Response response = given()
+                .contentType("application/x-www-form-urlencoded; charset=UTF-8")
+                .formParam("identifier", loginId)
+                .formParam("password", password)
+                .header("Origin", "qa.elevate-shikshagraha.shikshalokam.org")
+                .post("user/v1/admin/login");
+
+        if (response.getStatusCode() != 200) {
+            throw new RuntimeException("Admin login failed → " + response.getStatusCode());
+        }
+
+        // Extract Admin token from response
+        adminToken = response.jsonPath().getString("result.access_token");
+
+    }
+
+    //Log in as Admin use the token for Delete user API
+    public static void deleteUserFromAdmin()
+    {
+        //https://qa.elevate-apis.shikshalokam.org/user/v1/admin/deleteUser/3090
+
+        Response response = given()
+                .header("X-auth-token", adminToken)
+                .delete("user/v1/admin/deleteUser/"+User_ID);
+
+        if (response.getStatusCode() != 200) {
+            System.out.println(response.getStatusCode());
+            throw new RuntimeException("Delete User failed → " + response.getStatusCode());
+
```

Eat the exception here and log the details 
remove the sysouts